frontend/src/components/teams/team_profile.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Link } from '@reach/router';
+import * as safeStorage from '../../utils/safe_storage';
+import { cancelablePromise } from '../../utils/promise';
+import { fetchLocalJSONAPI,
+        pushToLocalJSONAPI } from '../../network/genericJSONRequest';
+import { Button } from '../button';
+import { AreaIcon } from '../svgIcons/area';
+
+export class TeamProfile extends React.Component{
+    tmTeamsPromise;
+    constructor(props) {
+        super(props);
+        this.state = {
+            team: [],
+            inviteSend: "",
+            isMember: false,
+            isAdmin: false,
+            username: safeStorage.getItem('username'),
+        };
+      }
+
+    componentDidMount = () => {
+        this.getTeam();
+        console.log(this.props);
+        console.log(this.state.username);
+    }
+
+    getTeam = () => {
+        this.tmTeamsPromise = cancelablePromise(fetchLocalJSONAPI('teams/' + this.props.team_id));
```

Comment:
That API request is returning 404, although the team exists on the DB and is listed on `teams/`